README.md

```diff
@@ -77,3 +80,23 @@ ansible-playbook -i galaxy/openshift-toolkit/custom-dashboards/.applier galaxy/o
 # Deploy MDT Tool
 ansible-playbook -i .applier/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml -e provision=false
 ```
+
+
+### Deploying Exporters
+
+To deploy the exporters, first setup the following environment variables and create the following secret:
+
+```
+export GITHUB_REPOS=redhat-cop/mdt-quickstart
+export GITHUB_USER=<your github user name>
+export GITHUB_TOKEN=<your github token>
+oc process -f templates/github-secret.yaml -p GITHUB_USER=${GITHUB_USER} -p GITHUB_TOKEN=${GITHUB_TOKEN} -p GITHUB_REPOS=${GITHUB_REPOS}  | oc apply -f-
```

Comment:
Environment Variable `PROJECTS` is optionally needed by the lead time exporter

Reply:
It is a comma separated list too

Reply:
I added a separate config map to handle this.